src/EndOfYearBanner/index.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactElement } from 'react';
+import { BrazeBannerComponent } from '../BrazeBannerComponent';
+import { StorybookWrapper } from '../utils/StorybookWrapper';
+import { knobsData } from '../utils/knobsData';
+import { coreArgTypes } from '../storybookCommon/argTypes';
+import { BrazeMessageProps } from '.';
+
+export default {
+    component: 'EndOfYearBanner',
+    title: 'Banner/EndOfYearBanner',
+    argTypes: {
+        ...coreArgTypes,
+        header: {
+            name: 'header',
+            type: { name: 'string', required: true },
+            description: 'Header text',
+        },
+        body: {
+            name: 'body',
+            type: { name: 'string', required: true },
+            description: 'Body text',
+        },
+        boldText: {
+            name: 'boldText',
+            type: { name: 'string', required: true },
```

Comment:
We're saying this is required here, but not checking for it in the `canRender` above. I'm not sure which is right (we could check with Gavin) but we should be consistent.

Reply:
Making it not required just for now.

Reply:
I've changed this to no longer be required here or by the can render. I think banner works okay without bold text so making it optional for now seems okay, but you're right we should check with Gavin.